com.github.Anuken.Mindustry.yml

```diff
@@ -0,0 +1,54 @@
+app-id: com.github.Anuken.Mindustry
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+  
+command: mindustry.sh
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+
+  # We don't have a gradle generator just yet, so we don't build from source
+  - name: mindustry-binary
+    buildsystem: simple
+    build-commands:
+        - install -D Mindustry.jar /app/bin/Mindustry.jar
+        - install -D mindustry.sh /app/bin/mindustry.sh
+        - install -D com.github.Anuken.Mindustry.desktop /app/share/applications/com.github.Anuken.Mindustry.desktop
+        - install -D com.github.Anuken.Mindustry.appdata.xml /app/share/metainfo/com.github.Anuken.Mindustry.appdata.xml
+        - install -D com.github.Anuken.Mindustry.png /app/share/icons/hicolor/512x512/apps/com.github.Anuken.Mindustry.png
+
+    sources:
+      - type: file
+        url: https://f-droid.org/repo/io.anuke.mindustry/en-US/icon_Eno3XvqCZUcHRm3eMjiUleAxgzLopPe6-hkI7BHx1lU=.png
+        dest-filename: com.github.Anuken.Mindustry.png
+        sha256: 127a375efa82654707466dde32389495e0318332e8a4f7bafa1908ec11f1d655
+        
+      - type: file
+        path: com.github.Anuken.Mindustry.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I've renamed the file to "metainfo". I hope that is sufficient.